### ISSUE TITLE
Trim verification input and course name when deleting a course

### ISSUE THREAD

**Issue by urbanecm** [WikiEducationFoundation/WikiEduDashboard#1990]
Hello, 

the Dashboard wants verification via inputting the course name when deleting. This is fine. However, a course can be called "Test" and "Test " (test and a space). For computers, those are two different strings. Most humans filling the input don't see the difference. I think that both input and course name should be trimmed at least when deleting. Maybe we can even disallow courses with space on the start or on the end as well, I see no sense with such courses. 

Thank you for considering this. 

Martin

**Comment by ragesoss** [comment]
How old was the "Test " course that you deleted?

We've been trimming course names client-side for a while now, but did not do so in the past. But I discovered that cloned courses don't get the names trimmed (fixed, but not yet deployed), and it's possible that there are other bugs that would allow that to happen that I don't know about yet.

I'll look into this next week.

**Comment by urbanecm** [comment]
I created a new test course (called Testovice/Test, still exists) and immediately after it I tried to delete it. So, it was a few of seconds. I sent "Test " instead of "Test" to the dashboard when prompted for verification and I think both "Test " and "Test" (or even " Test") should be accepted, as it don't make difference to the human trying to delete. 

That's one direction of needed trimming. You say that nowadays courses cannot have a space on the end, however, https://outreachdashboard.wmflabs.org/courses/Wikimedia_%C4%8CR/Editaton_v_Ceskych_Budejovicich__(podzim_2018) contained a space and per Czeva's information, it wasn't cloned. I do not have any other information and I cannot verify what she said. I can only say when deleting, I needed to include the space in the verification form as well.
